solr/solr-ref-guide/modules/getting-started/pages/tutorial-aws.adoc

```diff
@@ -140,7 +140,7 @@ $ java -version
 +
 [source,bash,subs="verbatim,attributes+"]
 # download desired version of Solr
-$ wget http://archive.apache.org/dist/lucene/solr/{solr-full-version}/solr-{solr-full-version}.tgz
+$ wget http://archive.apache.org/dist/solr/solr/{solr-full-version}/solr-{solr-full-version}.tgz
```

Comment:
```suggestion
$ wget http://archive.apache.org/dist/solr/{solr-full-version}/solr-{solr-full-version}.tgz
```

Reply:
@HoustonPutman and I have chatted elsewhere about the future layout of the distribution arvhice for Solr, and the proposal was to prefix all Solr releases with `solr/x.y.z` the same way as operator releases have `solr-operator/x.y.z`, and in the future, perhaps solr-exporter or other tools becomes its own artifact? So my edit was on purpose, but we can of course discuss whether solr's main artifacts live on the root level. Various ASF projects to this in different ways.